api/src/main/resources/workbench-api.yaml

```diff
@@ -5055,10 +5055,9 @@ definitions:
         type: string
       contactEmailFailure:
         description: >
-          Whether or not contact email could be added to verification list. No longer in use.
+          DEPRECATED. Whether or not contact email could be added to verification list. No longer in use.
```

Comment:
You should be able to add `deprecated: true` here so that it maps to `@Deprecated` in Java.